Source/WebCore/dom/Element.cpp

```diff
@@ -5129,6 +5150,22 @@ void Element::setAttributeStyleMap(Ref<StylePropertyMap>&& map)
     ensureElementRareData().setAttributeStyleMap(WTFMove(map));
 }
 
+void Element::ensureFormAssociatedCustomElement()
+{
+    ASSERT(is<HTMLElement>(*this));
```

Comment:
Can we use RELEASE_ASSERT instead?

Reply:
Sure, another option would be moving `ensureFormAssociatedCustomElement()` and friends to `HTMLMaybeCustom`, which is the only place they are / should be used, but to do that we need to declare it as `Element`'s `friend` class, and it kinda breaks consistency with other element rare data access...

Reply:
Changed to `RELEASE_ASSERT(is<HTMLMaybe` like discussed on Slack.